src/app/shared-services/inferences/engine/staypoint-engine/index.ts

```diff
@@ -0,0 +1,106 @@
+import { Inference } from 'src/app/model/inference'
```

Comment:
Like above: Why `index.ts` rather than `staypoint-engine.ts`

Reply:
same as above, I renamed it.